### DISCUSSION TITLE
algebra relazionale - 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#59] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/77b0f9ea-b789-47fd-9668-ec7e88cf7a54)

**Comment by 7Andrei** [comment]
![AlgebraRelazionale2](https://github.com/sapienzastudentsnetwork/bdi2324/assets/97444723/b87239d5-bd66-4a9a-b73b-08245acc77a2)

**Comment by Carlo-Armato** [comment]
![2](https://github.com/sapienzastudentsnetwork/bdi2324/assets/158755407/0f37481a-628e-43c7-920c-0956d66b0035)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/ed9b424f-1799-4e0b-be51-540f277c8da0)
